stable/xray/values.yaml

```diff
@@ -100,6 +100,24 @@ xray:
     enabled: false
     # certificateSecretName:
 
+  # Support for open metrics is only available for Artifactory 7.7.x (appVersions) and above.
+  # To enable set `.Values.xray.openMetrics.enabled` to `true`
+  # Refer - https://www.jfrog.com/confluence/display/JFROG/Open+Metrics
+  openMetrics:
+    enabled: true
+    ## Settings for pushing metrics to Insight - enable filebeat to true
+    filebeat:
+      enabled: false
+      log:
+        enabled: false
+        ## Log level for filebeat. Possible values: debug, info, warning, or error.
+        level: "info"
```

Comment:
I'm not sure about this section because I didn't see it on : https://www.jfrog.com/confluence/display/JFROG/Xray+System+YAML